zkevm-circuits/src/evm_circuit/execution/coinbase.rs

```diff
@@ -91,16 +91,21 @@ impl<F: Field> ExecutionGadget<F> for CoinbaseGadget<F> {
 mod test {
     use crate::test_util::run_test_circuits;
     use eth_types::bytecode;
+    use mock::TestContext;
 
-    fn test_ok() {
+    #[test]
+    fn coinbase_gadget_test() {
         let bytecode = bytecode! {
             COINBASE
             STOP
         };
-        assert_eq!(run_test_circuits(bytecode), Ok(()));
-    }
-    #[test]
-    fn coinbase_gadget_test() {
-        test_ok();
+
+        assert_eq!(
+            run_test_circuits(
+                TestContext::<2, 1>::simple_ctx_with_bytecode(bytecode).unwrap(),
```

Comment:
I have to say that it's a bit strange specifying the `::<2,1>` because the `simple_ctx_with_bytecode` is not generic and doesn't require `self`.  But without it, the code doesn't compile.  So I don't have any solution for this :P

Reply:
It surprised me a lot too!! It's a weird `const_generics` property that I did not see before.